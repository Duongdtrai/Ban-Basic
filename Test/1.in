37
24
59 74 31 82 9 28 47 30 51 63 89 75 34 77 24 30 49 71 44 33 12 66 55 98 
68
13 65 27 60 34 43 78 5 69 2 3 26 94 41 83 69 59 14 96 60 91 78 55 3 87 42 24 65 97 84 45 91 50 80 76 72 2 1 61 1 24 87 21 30 52 84 30 23 55 31 91 42 52 56 36 63 83 43 25 15 35 96 3 60 8 14 67 61 
94
69 3 84 11 42 68 77 55 79 37 95 26 15 80 53 99 1 88 77 14 34 94 58 88 19 95 24 35 91 63 79 47 75 88 56 72 74 89 44 64 97 28 80 40 89 83 45 40 23 61 51 13 70 82 75 25 33 22 54 33 6 75 30 45 57 6 37 61 60 84 23 3 68 14 73 68 11 95 81 44 65 83 41 44 66 84 96 67 91 82 100 55 78 65 
60
60 77 70 62 29 84 27 95 5 86 25 7 33 7 22 100 43 12 69 97 89 20 83 53 72 31 14 24 68 7 50 74 64 78 71 81 12 10 8 19 88 92 16 33 19 12 99 46 71 90 93 67 77 52 56 62 86 58 77 17 
49
21 27 48 80 17 76 8 24 71 68 34 50 33 3 34 100 45 63 16 25 82 91 58 5 77 10 58 15 1 5 40 49 33 83 71 49 24 22 88 46 4 91 94 7 5 29 23 15 41 
15
25 8 52 17 10 2 14 50 52 7 95 91 95 11 7 
72
32 79 100 53 6 14 26 71 7 41 12 3 68 72 97 71 81 56 95 52 34 44 100 28 59 34 11 41 55 35 23 26 77 24 50 11 72 1 42 5 39 10 89 81 44 74 64 53 71 80 47 84 3 64 23 3 61 80 48 57 6 80 84 52 14 16 99 33 92 65 28 75 
94
53 15 69 74 17 74 79 98 18 21 44 58 22 28 93 40 71 69 89 6 64 80 25 2 61 23 69 59 99 45 78 33 100 83 45 86 16 16 93 32 30 77 43 11 53 8 19 17 34 82 67 26 4 45 30 90 67 12 26 30 98 23 7 51 62 41 13 22 89 10 59 48 89 29 23 60 92 34 66 52 52 45 77 43 65 63 17 72 69 68 10 7 37 50 
72
65 68 33 2 72 65 50 74 68 94 40 89 98 10 78 39 51 78 30 93 52 58 38 82 58 52 29 44 65 57 71 78 42 21 56 7 35 93 25 20 7 41 14 7 2 93 19 36 79 66 60 41 95 40 57 93 37 82 61 70 66 36 4 42 67 67 36 100 94 11 15 80 
73
65 66 84 81 55 61 83 11 19 82 94 75 89 54 27 37 13 38 30 86 1 88 86 21 62 97 86 18 50 9 56 97 44 57 1 2 99 7 13 79 87 88 51 49 100 98 50 1 28 22 79 87 53 64 21 58 24 55 95 3 24 11 67 7 14 52 86 23 49 6 25 80 11 
82
43 70 18 76 23 20 64 91 46 69 37 89 17 64 67 79 16 68 33 9 61 35 53 40 25 55 54 67 86 36 26 14 37 71 61 11 26 67 72 46 65 95 52 23 28 6 72 87 93 95 59 19 43 92 72 92 72 89 3 26 3 16 93 76 12 56 19 55 14 50 8 60 48 49 34 77 95 75 73 16 28 58 
20
91 13 6 49 78 16 53 82 68 52 87 86 86 34 15 20 29 86 72 58 
38
7 81 9 65 67 86 80 45 73 11 86 94 57 65 92 25 40 73 51 57 40 23 74 40 78 56 27 46 59 44 81 71 59 95 100 27 20 17 
17
26 5 18 49 7 49 84 58 94 12 61 21 66 46 95 99 31 
58
1 58 3 86 17 58 11 68 68 75 86 2 14 67 41 68 85 64 63 61 91 82 16 62 44 70 84 61 8 37 49 86 61 18 68 64 64 47 66 76 43 30 91 55 37 35 50 59 30 39 52 38 11 59 44 59 46 9 
98
39 40 98 58 71 83 58 40 80 17 99 51 96 92 61 4 31 69 95 63 92 28 80 70 15 67 46 47 16 16 92 35 64 11 45 7 59 15 48 4 29 48 28 48 7 35 33 43 39 54 37 18 51 38 32 7 57 19 85 1 30 30 42 44 23 33 66 59 86 84 70 62 33 8 23 100 19 76 9 14 62 60 83 60 23 69 8 48 44 78 82 55 92 100 57 7 9 12 
54
24 7 51 93 58 70 11 77 100 98 52 20 57 43 41 88 35 37 38 50 5 37 71 34 80 26 91 57 8 92 69 78 69 12 36 85 99 98 39 13 86 81 30 48 70 47 40 72 39 16 12 14 6 100 
64
6 60 2 28 17 52 32 100 19 13 33 75 4 60 92 19 32 82 31 40 89 97 20 38 40 32 68 50 59 44 39 66 60 83 34 71 89 84 7 82 18 29 73 12 59 91 8 80 2 58 30 39 22 12 22 64 7 99 10 49 73 50 39 11 
98
74 88 43 44 48 46 58 45 1 71 90 79 5 79 98 33 30 49 4 22 7 19 36 32 100 92 98 57 47 62 76 84 44 12 84 26 69 38 88 43 75 97 79 27 50 28 56 51 38 39 86 54 70 78 22 16 14 22 55 23 54 83 9 72 19 69 37 97 12 1 64 99 71 89 34 28 53 42 78 31 76 61 25 92 23 36 65 77 12 9 15 90 9 70 57 71 47 83 
34
56 93 86 2 91 67 61 44 92 31 29 52 3 27 27 46 30 46 28 21 50 86 75 56 29 50 78 56 100 18 68 54 43 50 
43
51 75 57 4 11 10 70 29 85 69 76 70 68 22 60 63 18 9 42 64 64 95 21 33 83 19 83 29 87 53 20 1 36 20 70 31 69 66 5 75 96 76 42 
96
56 78 35 79 16 48 40 95 92 21 79 83 23 98 9 86 55 75 97 99 93 25 40 15 23 50 15 50 45 39 78 94 82 49 25 99 44 33 54 100 87 81 87 18 63 22 51 40 16 38 36 51 41 52 41 43 70 27 95 20 4 7 43 51 98 95 56 39 39 29 43 53 20 23 60 84 69 38 34 24 89 23 56 9 86 36 3 32 52 96 10 51 57 90 2 11 
56
56 87 32 91 73 39 45 81 60 38 80 97 96 18 37 86 22 24 63 85 56 42 19 78 46 85 55 11 90 65 22 87 25 51 81 59 39 11 66 98 78 53 38 89 30 11 75 82 41 39 16 77 25 76 87 1 
53
99 25 64 41 90 13 91 35 35 69 88 95 25 60 24 50 65 87 90 59 76 84 27 54 73 21 14 6 90 10 69 84 93 44 59 42 84 99 94 48 99 25 12 70 13 42 55 31 93 79 67 37 27 
16
17 26 70 55 14 6 26 69 93 68 29 44 68 8 96 64 
30
68 34 51 45 70 14 74 98 40 23 80 29 74 39 82 20 70 88 55 60 74 76 36 98 92 8 94 3 4 55 
10
15 70 89 64 76 39 47 46 55 93 
88
31 58 59 88 47 37 74 11 35 95 64 28 26 66 53 90 5 52 52 82 35 47 38 44 71 61 71 46 24 19 37 53 91 49 35 64 40 77 19 36 61 91 94 96 58 22 15 52 42 10 96 99 92 68 77 18 12 75 53 28 34 98 71 77 57 3 3 82 83 59 68 37 3 64 93 52 32 14 40 94 72 55 90 65 64 97 77 89 
17
23 43 87 35 4 63 83 47 92 47 85 36 84 41 58 35 67 
29
65 28 57 98 37 86 19 68 8 55 70 24 99 61 57 46 52 79 12 6 42 75 80 84 49 76 25 48 95 
62
74 35 68 32 29 37 38 77 12 17 35 30 88 91 78 79 43 56 88 8 89 93 8 75 84 45 85 15 38 34 88 40 40 38 96 89 7 18 21 79 62 27 77 7 77 40 70 78 64 46 22 88 5 32 67 66 85 5 85 33 27 87 
96
15 44 43 98 6 22 72 11 29 71 80 37 99 32 54 13 59 17 50 33 55 99 11 48 61 18 51 6 47 4 53 25 29 15 98 15 94 51 99 42 35 78 68 99 84 54 28 61 46 79 69 7 79 18 27 55 73 1 66 67 59 42 3 33 28 41 40 16 54 7 27 21 41 13 13 81 76 33 20 63 15 21 54 72 54 34 14 6 79 69 63 88 45 58 40 81 
18
61 46 84 37 43 37 76 54 84 86 94 58 31 89 77 87 65 21 
94
54 64 1 70 57 58 71 26 97 37 4 100 68 48 76 24 59 66 38 41 42 25 86 5 77 41 10 98 87 42 54 42 73 50 14 45 77 46 53 84 29 26 99 88 35 3 100 33 33 55 1 68 16 22 37 98 35 26 31 41 63 51 48 67 100 54 86 73 38 18 88 47 10 72 78 17 98 11 50 70 58 27 76 16 73 79 100 32 12 89 35 31 88 80 
91
10 17 53 61 61 97 48 3 87 45 89 97 32 80 1 100 89 73 70 83 49 22 21 29 5 29 12 1 69 39 24 83 96 77 89 25 28 85 31 24 13 94 1 43 42 12 25 95 67 5 16 12 26 50 48 15 23 58 11 40 78 4 40 22 97 73 52 8 16 92 25 2 4 13 15 6 2 41 42 82 85 32 77 23 19 16 93 86 15 86 15 
38
38 17 34 89 74 1 30 77 8 24 53 52 72 52 91 96 28 56 71 14 29 15 84 83 79 37 92 39 61 56 49 27 96 73 6 53 39 66 
61
80 40 90 66 99 100 1 8 48 100 96 69 22 8 45 12 8 46 47 99 73 6 11 14 12 54 39 7 98 3 66 20 16 98 90 58 46 78 23 21 24 87 81 31 34 56 94 71 67 53 25 41 84 45 96 96 40 47 42 98 17 
