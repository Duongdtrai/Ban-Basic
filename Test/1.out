48961176
3010275750
8055451920
2275774712
574750852
767472
2052090832
5301485224
4318682096
309362700
6984255660
13995486
373496928
4889100
1696430340
11808455386
1325670230
1005683916
2428754910
201619572
512658384
2362716950
1673598160
734769168
5077872
11618280
486416
6005332850
13788054
13095380
2459527228
10961194874
24249998
2233721820
1765571356
358612744
1155543840
